docs/platforms/php/common/data-management/data-collected.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Data Collected
+description: "See what data is collected by the Sentry SDK."
+sidebar_order: 1
+---
+
+Sentry takes data privacy very seriously and has default settings in place that prioritize data safety, especially when it comes to personally identifiable information (PII) data. When you add the Sentry SDK to your application, you allow it to collect data and send it to Sentry during the runtime of your application.
+
+The category types and amount of data collected vary, depending on the integrations you've enabled in the Sentry SDK. Here's a list of data categories the Sentry PHP SDK collects:
+
+## HTTP Headers
+
+By default, the Sentry SDK doesn't send any HTTP headers.
+
+To start sending HTTP headers, set the [`send_default_pii` option](../configuration/options/#send-default-pii) to `true`.
```

Comment:
This is a little bit contradicting. 

> By default, the Sentry SDK doesn't send any HTTP headers.

I interpret is as `By default, there are no PII (headers)`.  
Then, there is `default_pii` that sends headers.

Reply:
Yeah, we can reword this. The SDK making a request to a Sentry instance indeed has HTTP headers attached, but we ofc are talking about the payload containing headers here.